atomic_reactor/plugins/exit_remove_worker_metadata.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from atomic_reactor.plugin import ExitPlugin
+from atomic_reactor.constants import PLUGIN_REMOVE_WORKER_METADATA_KEY
+
+__all__ = ('RemoveWorkerMetadataPlugin', )
+
+
+def defer_removal(workflow, cf_map, osbs):
```

Comment:
This function should be moved to `utils/` as its being used in a different plugin - or just to `atomic_reactor/plugins/post_fetch_worker_metadata.py`

Reply:
You think? It works like this for remove_built_image.

Reply:
Keeping this function here seems reasonable to me.